commit a3cd71de3bed3c7870048301968222d6c05108d4 (refs/changes/73/873/1)
Author: Angela Fong <angela.fong@joyent.com>
Date:   2016-11-10T09:54:45-08:00 (2 years, 11 months ago)
    
    ADMINUI-2354 allow ufds_master credentials to differ from local ufds

diff --git a/sapi_manifests/adminui/template b/sapi_manifests/adminui/template
index 991d7b45..b46d43a2 100644
--- a/sapi_manifests/adminui/template
+++ b/sapi_manifests/adminui/template
@@ -9,7 +9,12 @@
 	"ufdsMaster": {
 		"url": "ldaps://{{^ufds_remote_ip}}{{{ufds_domain}}}{{/ufds_remote_ip}}{{{ufds_remote_ip}}}",
 		"bindDN": "{{{ufds_ldap_root_dn}}}",
+		{{^ufds_remote_ldap_root_pw}}
 		"bindPassword": "{{{ufds_ldap_root_pw}}}",
+		{{/ufds_remote_ldap_root_pw}}
+		{{#ufds_remote_ldap_root_pw}}
+		"bindPassword": "{{{ufds_remote_ldap_root_pw}}}",
+		{{/ufds_remote_ldap_root_pw}}
 		"clientTimeout": 240000,
 		"cache": {
 			"size": 5000,
